source/type/pb_converter.cpp

```diff
@@ -871,7 +871,10 @@ void PbCreator::Filler::fill_pb_object(const nt::Route* r, pbnavitia::Route* rou
         return;
     }
 
-    fill(r->line, route);
+    copy(0, dump_message_options).fill(r->line, route);
+    if ((depth > 0) && (!this->pb_creator.disable_geojson)) {
+        fill(&r->line->shape, route->mutable_line());
```

Comment:
:heavy_check_mark: In case someone wants to change that in the **future** it's possible (but not done in this PR).


The code here works because shape doesn't have sub-object, but it would be more consistent to do:
```suggestion
        copy(0, dump_message_options).fill(&r->line->shape, route->mutable_line());
```

Reply:
:+1: